src/ResourceManager/Network/Commands.Network/help/Remove-AzureRmVpnClientIpsecParameter.md

```diff
@@ -0,0 +1,186 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/remove-azurermvpnclientipsecparameter
+schema: 2.0.0
+---
+
+# Remove-AzureRmVpnClientIpsecParameter
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+### ByFactoryName (Default)
+```
+Remove-AzureRmVpnClientIpsecParameter -VirtualNetworkGatewayName <String> -ResourceGroupName <String> [-Force]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByFactoryObject
+```
+Remove-AzureRmVpnClientIpsecParameter -InputObject <PSVirtualNetworkGateway> [-Force]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Remove-AzureRmVpnClientIpsecParameter -ResourceId <String> [-Force] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Virtual Network Gateway is the object representing your gateway in Azure.
+
+The **Remove-AzureRmVpnClientIpsecParameter** cmdlet removes the vpn custom ipsec parameters set on your Virtual Network Gateway, which in turn sets default vpn ipsec policy on VPN gateway based on VirtualNetworkGateway Name and Resource Group Name passed.
+
+## EXAMPLES
+
+### 1: Deletes the set vpn ipsec parameters set on the Virtual Network Gateway
+```
+Remove-AzureRmVpnClientIpsecParameter -VirtualNetworkGatewayName myGateway -ResourceGroupName myRG
```

Comment:
@Nilambari same comment about adding the corresponding output

Reply:
Done.